model.py

```diff
@@ -17,8 +17,9 @@ def __init__(self, config):
         super().__init__()
         self.config = config
         self.embed_dim = config.hidden_size
-        self.CLSLayer = nn.Embedding(1, self.embed_dim)
+        self.CLSLayer = nn.Embedding(2, self.embed_dim)
```

Comment:
@SunCreation
minor, how about change `CLSLayer` to `SpecialLayer`??
Even if it is not `SpecialLayer`, it would be good if it is a word that contains two meanings, `CLS` and `SEP`.

Reply:
@YejinHwang909 
Oh, thanks!!!!
I'll apply that right away!